src/components/DropdownList/DropdownList.tsx

```diff
@@ -0,0 +1,33 @@
+import { Person } from '../../types/Person';
+import './DropdownList.scss';
+
+type Props = {
+  peoples: Person[] | null;
```

Comment:
The prop 'peoples' might be better named as 'people', as 'people' is already plural.